srcpkgs/cloud-utils/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'cloud-utils'
+pkgname=cloud-utils
+version=0.33
+revision=1
+archs="x86_64"
```

Comment:
why only x86_64?

Reply:
i did not test for other arch.just verified its functionality on x86_64.

Reply:
> Should specify its dependencies.
> We do not have cdrkit packaged.
I will check these.

Reply:
https://github.com/archlinux/svntogit-community/blob/packages/cloud-utils/trunk/PKGBUILD  
this doesn't have any dependencies. They are a bunch of shell scripts.

Reply:
> They are a bunch of shell scripts.

not exactly: https://github.com/canonical/cloud-utils/blob/main/bin/ec2metadata

Reply:
> this doesn't have any dependencies

yes it does: https://github.com/archlinux/svntogit-community/blob/packages/cloud-utils/trunk/PKGBUILD#L33